modules/cluster-logging-view-cluster-dashboards.adoc

```diff
@@ -0,0 +1,4 @@
+[id="cluster-logging-view-cluster-dashboards-about_{context}"]
+= About viewing cluster dashboard
+
+The {product-title} Logging dashboard contains charts that show details about your Elasticsearch instance at a cluster-level that you can use to diagnose and anticipate problems.
```

Comment:
```suggestion
The {product-title} Logging dashboard contains charts that show details about your Elasticsearch instance at a cluster level that you can use to diagnose and anticipate problems.
```

This might be clearer if you break it up. Perhaps something like:

The {product-title} Logging dashboard contains charts that show details about your Elasticsearch instance at the cluster level. These charts help you diagnose and anticipate problems.

Reply:
acked